game-core/src/system/camera.rs

```diff
@@ -0,0 +1,48 @@
+use amethyst::renderer::Camera; 
+use amethyst::{
+    core::Transform,
+    ecs::{Join, ReadStorage, System, WriteStorage},
+};
+
+use crate::component::Player;
+
+pub struct Movement;
+
+impl<'s> System<'s> for Movement {
+    type SystemData = (
+        ReadStorage<'s, Player>,
+        ReadStorage<'s, Camera>,
+        WriteStorage<'s, Transform>,
+    );
+
+    fn run(&mut self, (players, cameras, mut transforms): Self::SystemData) {
+        let mut player_x: f32 = 0.0;
+        let mut player_y: f32 = 0.0;
+
+        // let mut camera_x: f32 = 0.0;
+        // let mut camera_y: f32 = 0.0;
+
+        // let mut camera_safe_x: f32 = 0.0;
+        // let mut camera_safe_y: f32 = 0.0;
+
+        for (_, transform) in (&players, &mut transforms).join() {
+            player_x = transform.translation.x - 256.0 as f32;// * 5.0;
+            player_y = transform.translation.y - 256.0 as f32;// * 5.0;
+        }
```

Comment:
I think I want to standardize how we do this because this is the 3rd version of essentially unwrapping a player transform we've done. I'm not gonna ask you to do it in this PR but ill open an issue and go back and standardize how we do this. Its going to boil down to either using Options, if we expect the player to possibly not be initialized, or unwrapping.

I would like to make it an assumption that there's only ever one Player.